src/App.js

```diff
@@ -6,6 +6,9 @@ import GamePage from './components/GamePage'
 
 export default function App() {
   const [players, setPlayers] = useState([])
+  // type players = {
```

Comment:
If you don't use it, then delete the comments :)
I think the `type` for TypeScript will have to be declared outside of the component, like PropTypes. And The type for players is an array and I don't know, but before using TypeScript you need to do some configurations. Not so sure about that though.